src/receivers/HTTPResponseAck.ts

```diff
@@ -42,15 +47,33 @@ export class HTTPResponseAck implements ResponseAck {
     this.isAcknowledged = false;
     this.processBeforeResponse = args.processBeforeResponse;
     this.unhandledRequestHandler = args.unhandledRequestHandler ?? httpFunc.defaultUnhandledRequestHandler;
+    this.unhandledFunctionRequestTimeoutMillis = 10001;
```

Comment:
Since we might want to change the timeout in webapp, do we want to start with something smaller like 5sec?